integration_tests/wasm/mono_pkg.yaml

```diff
@@ -18,6 +18,6 @@ stages:
       - which google-chrome-beta
       os:
         - linux
-    - test:
+    - test: --timeout=60s --retry=2
```

Comment:
Note that you actually can't configure the number of retries here (its just a flag to enable or disable retries). The number of retries can be configured in dart_test.yml though.